### PR TITLE
Configure formatter to sort imports

### PR DESCRIPTION
**Pull request by Julian702**

### CODE REVIEW

File: frontend/tsconfig.json [hunk 1]
```diff
@@ -13,7 +13,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "paths": {
+      "@/*": ["./*"]
+    },
```
Comment (darmiel) [inline frontend/tsconfig.json:19]: is this required?

**Review by darmiel — APPROVED** [review]

File: frontend/tsconfig.json (hunk 1 above)
Comment (Julian702) [reply frontend/tsconfig.json:19]: yes

Comment (darmiel) [reply frontend/tsconfig.json:19]: great

File: frontend/.prettierrc [hunk 2]
```diff
@@ -1,4 +1,15 @@
 {
-  "tabWidth": 4,
-  "useTabs": true
+	"tabWidth": 4,
+	"useTabs": true,
+	"singleQuote": true,
```
Comment (darmiel) [inline frontend/.prettierrc:4]: Kioku uses double quotes, doesn't it?

Comment (Julian702) [reply frontend/.prettierrc:4]: Most of the time we used double quotes, but it was not consistent, so I have configured it here now.

**Review by TomRomeo — APPROVED** [review]
ρ

**Pull request merged**